src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1072,16 +1072,13 @@ components:
         well_known_addresses:
           type: object
           required:
-            - account_package
             - faucet
             - epoch_manager
             - clock
             - ecdsa_secp256k1
             - eddsa_ed25519
             - xrd
           properties:
-            account_package:
-              $ref: "#/components/schemas/EntityAddress"
             faucet:
               $ref: "#/components/schemas/ComponentAddress"
             epoch_manager:
```

Comment:
I'm quite sure we should also change NFID Type `Number` to `Integer`.

Reply:
done.